swagger.yaml

```diff
@@ -4101,6 +4103,7 @@ definitions:
       base_dns_domain:
         type: string
         description: Base domain of the cluster. All DNS records must be sub-domains of this base and include the cluster name.
+        pattern: '^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$'
```

Comment:
Please consult Avishai if we do not need to extract base_dns_domain to the description part and use ref here

Reply:
We decided to remove the swagger changes altogether.
Please see description.